### DISCUSSION TITLE
v1.16.0

### DISCUSSION THREAD

**Discussion by Victor-IX** [Victor-IX/Blender-Launcher-V2#8] [Announcements]
# What's Changed

## ⚠Info:
- **Blender Foundation is currently blocking the request on the /release website for the stables versions making it impossible to retrieve those versions, this issue will be fixed during the week**

## 💡Feature:
- Using Blender's JSON API for gathering builds available from builder.blender.org
## ⚙️Other:
- Linux Version is now available

<hr /><em>This discussion was created from the release <a href='https://github.com/Victor-IX/Blender-Launcher-V2/releases/tag/v1.16.0'>v1.16.0</a>.</em>

**Comment by eisenbruch** [comment]
When I open it I get a windows error and I have to click "run anyway" and then once it launches it still shows v1.15.3 with a prompt to update to 1.16.0

**Reply by Victor-IX** [reply]
Ah, I didn't push the right build 😅
it's fix sorry about that

**Reply by Victor-IX** [reply]
For the windows protected you pc warning, IDK I need to take a look how windows program credential works for that

**Reply by eisenbruch** [reply]
now it shows im on 1.16, thanks.

hope you resolve the windows warning. it now takes 3 clicks just to open blender launcher

**Reply by zeptofine** [reply]
Yyyyeahh this is a byproduct of our binary building system using pyinstaller. It's a pretty big issue due to pyinstaller basically making a zip file of all the necessary stuff and extracting it every time you launch

**Reply by Victor-IX** [reply]
I have the warning only once on my side, the first time I launch the app
